### PR TITLE
CB -> LB: Create Basic UI for Activities Area of Lesson Plan

### PR DESCRIPTION
**Pull request by dmcavoy**
Recreates some very basic UI components from Curriculum Builder for displaying activities in a Lesson Plan. Adds a sample activity to the Lesson Overview page as a place holder for right now until the activities model is created.  This UI is also going to be used to preview the activities in the activities editor (PR to follow).

Screen Shot 2020-09-05 at 10 46 27 PM

## Testing story

Added unit tests for new UI components.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/lessonOverview/activities/Activity.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  };
+
+  render() {
+    const {activity} = this.props;
+
+    return (
+      <div>
+        <h2 style={styles.activityHeader}>{`${activity.displayName} (${
+          activity.time
+        } minutes)`}</h2>
```
Comment (davidsbailey) [inline apps/src/templates/lessonOverview/activities/Activity.jsx]: I think we will need to translate these, so please extract a translatable string here

File: apps/src/templates/lessonOverview/activities/ProgressionDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  static propTypes = {
+    progression: PropTypes.object
+  };
+
+  render() {
+    const {progression} = this.props;
+
+    return (
+      <div style={styles.progressionBox}>
+        <ProgressLevelSet
```
Comment (davidsbailey) [inline apps/src/templates/lessonOverview/activities/ProgressionDetails.jsx:25]: nice reuse of this component!

File: apps/test/unit/templates/lessonOverview/LessonOverviewTest.js [hunk 3]
```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import {shallow} from 'enzyme';
 import {expect} from '../../../util/reconfiguredChai';
 import LessonOverview from '@cdo/apps/templates/lessonOverview/LessonOverview';
+import {sampleActivities} from '../../../../test/unit/lib/levelbuilder/lesson-editor/activitiesTestData';
```
Comment (davidsbailey) [inline apps/test/unit/templates/lessonOverview/LessonOverviewTest.js]: nit: here and below, you shouldn't have to go up quite so many directories to find this: `import {sampleActivities} from '../..//lib/levelbuilder/lesson-editor/activitiesTestData';` but if this helps you or your IDE somehow then it seems ok to leave.

**Review by davidsbailey — APPROVED** [review]
Nice work Dani, this is shaping up well!

**Review by uponthesun — APPROVED** [review]

File: apps/src/templates/lessonOverview/activities/Activity.jsx (hunk 1 above)
Comment (dmcavoy) [reply apps/src/templates/lessonOverview/activities/Activity.jsx]: Good call. I translated everything that was on the lesson plan page so far as well.

**Pull request merged**